mule-extensions-api/src/main/java/org/mule/runtime/extension/api/manifest/DescriberManifest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.manifest;
+
+import org.mule.runtime.extension.api.introspection.declaration.spi.Describer;
+import org.mule.runtime.extension.api.manifest.ExtensionManifestBuilder.DescriberManifestBuilder;
+
+import java.util.Map;
+
+/**
+ * Manifest that enunciates the main properties of a {@link Describer}
+ * to be instantiated by the Mule Runtime.
+ * <p>
+ * Instances should be created through the {@link DescriberManifestBuilder}
```

Comment:
I don't understand why you should provide an API implementation of those classes in that case. The point of defining an interface is that everyone can provide an implementation of it, you are separating concerns. If you can only use instances created from the builder, then seems to me that this should be an abstract class